db/index.js

```diff
@@ -37,16 +38,20 @@ const benuImageURLs = [
 ];
 
 benuImageURLs.forEach((url) => {
-  const image = new Image({ url });
+  const image = new Image({
+    url,
+    restaurantId: 1
```

Comment:
you don't need name, type?

Reply:
@ez-li This was based on an image from Benu, which had no extra information displayed aside from the image. When I seed my database later, I'll be sure to have a variety of types/names/photographers